htdocs/margin/tabs/productMargins.php

```diff
@@ -144,7 +144,7 @@
             if (!$user->rights->societe->client->voir && !$socid) $sql.= ", ".MAIN_DB_PREFIX."societe_commerciaux as sc";
             $sql.= " WHERE f.fk_soc = s.rowid";
             $sql.= " AND f.fk_statut > 0";
-            $sql.= " AND s.entity = ".$conf->entity;
+            $sql.= " AND f.entity = ".$conf->entity;
```

Comment:
@mapiolca @eldy 
better is : 

$sql.= " AND s.entity IN (".getEntity('thirdparty').");
$sql.= " AND f.entity IN (".getEntity('invoice').");